.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+        branches:
+            - master
```

Comment:
Can we remove the restriction of only running on `master`?
This should be run on all, or at least some special branches
@yyoncho

Reply:
make sense - let's run it on top of everything in emacs-lsp